gap/ClassicalNormalizerMatrixGroups.gi

```diff
@@ -8,14 +8,13 @@ function(d, q)
 
     F := GF(q);
     zeta := PrimitiveElement(F);
-    A := Sp(d, q).1;
-    B := Sp(d, q).2;
+    AandB := ShallowCopy(GeneratorsOfGroup(Sp(d, q)));
```

Comment:
When reading this code later on, nobody will know about A and B, and thus wonder about this name.

I would recommend calling the variable `gens` instead (you could use `generators`, too, but in the GAP group theory code, the use of `gens` for a set of generators is ubiquitous). Then, later, replace `result := Group(Concatenation(AandB, [C]));` by
```gap
   Add(gens, C);
   result := Group(gens);
```
resp. replace `result := Group(Concatenation(AandB, [C, E]));` by
```gap
   Append(gens, [C, E]);
   result := Group(gens);
```